examples/java-test-repo/pom.xml

```diff
@@ -10,7 +10,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <sonar.language>java</sonar.language>
-    <sonar.host.url>http://sonar-test.lab.eng.rdu2.redhat.com</sonar.host.url>
+    <sonar.host.url>http://sonar-server.lab.eng.rdu2.redhat.com</sonar.host.url>
```

Comment:
any chance to use environment variable here? or update it during job run run?

Reply:
Do you asking because you want to prevent exposing internal URL?